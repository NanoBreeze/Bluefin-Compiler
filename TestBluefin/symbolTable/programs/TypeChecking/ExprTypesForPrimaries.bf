int a;
bool b;
float c;
string d;

int main() {
	a;
	b;
	c;
	d;
	int f;
	f;
	true;
	false;
}