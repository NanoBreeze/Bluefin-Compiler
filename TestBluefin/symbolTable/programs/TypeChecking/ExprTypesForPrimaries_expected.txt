a - t_int
b - t_bool
c - t_float
d - t_string
f - t_int
true - t_bool
false - t_bool
